contracts/Controller.sol

```diff
@@ -27,11 +27,11 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     using MarginAccount for MarginAccount.Vault;
     using SafeMath for uint256;
 
-    AddressBookInterface public addressbook;
-    WhitelistInterface public whitelist;
-    OracleInterface public oracle;
-    MarginCalculatorInterface public calculator;
-    MarginPoolInterface public pool;
+    AddressBookInterface internal addressbook;
+    WhitelistInterface internal whitelist;
+    OracleInterface internal oracle;
+    MarginCalculatorInterface internal calculator;
+    MarginPoolInterface internal pool;
```

Comment:
hmm imo still valuable to have these as public. During upgrades it could get confusing if we can't see the state changes properly.

Reply:
@antoncoding Can u add a getter for those ? like `getConfiguration()`